extensions/sherlock-rxjs/rxjs.ts

```diff
@@ -17,17 +17,17 @@ export function toObservable<V>(derivable: Derivable<V>, options?: Partial<React
     });
 }
 
-export function fromObservable<V>(observable: Subscribable<V>): Derivable<V> {
+export function fromObservable<V>(observable: Observable<V>): Derivable<V> {
     const atom$ = atom.unresolved<V>();
 
-    let subscription: Unsubscribable | undefined;
+    let subscription: Subscription | undefined;
```

Comment:
V5 does not know Unsubscribable, this was introduced in V6. Note that it does not break the code as the used method unsubscribe() existed also in V5.